.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,70 @@
+name: "🐛 Bug report"
+description: Report a bug found while using the Berty app.
+body:
```

Comment:
I think you should add something like this at the top of the file to avoid duplicate as much as possible.

```
- type: checkboxes
  attributes:
    label: Is there an existing issue for this?
    description: Please search to see if an issue already exists for the bug you encountered.
    options:
    - label: I have searched the existing issues
      required: true
```

Reply:
Then you should add a dropdown with the following entries:

```
- type: dropdown
    id: product
    attributes:
      label: Berty product
      description: What Berty product are you seeing the problem on?
      options:
        - Mobile app
        - Desktop app
        - CLI tools (berty mini, daemon, etc.)
        - Code package 
- type: input
    id: product-version
    attributes:
      label:  Berty product version
      description: What is the version of this Berty product?
      placeholder: |
      You can find the version of the mobile and desktop apps in the settings.
      To display the CLI tool version, run `berty version`.
    validations:
      required: true
```

Reply:
Nope, you did the right thing! 👍